publish.prx.org/pipeline.yml

```diff
@@ -0,0 +1,218 @@
+                  - "ecr:PutImage"
+                # TODO maybe make name a template config param?
+                Resource: "arn:aws:ecr:us-east-1:561178107736:repository/publish.prx.tech"
+              - Effect: Allow
+                Action:
+                  - "ecr:GetAuthorizationToken"
+                Resource: "*"
+        - PolicyName: ArtifactStoreBasicAccess
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "s3:PutObject"
+                  - "s3:GetObject"
+                  - "s3:GetObjectVersion"
+                Resource: !Sub arn:aws:s3:::${ArtifactStore}/*
+        - PolicyName: TemplateConfigStoreBasicAccess
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "s3:PutObject"
+                  - "s3:GetObject"
+                  - "s3:GetObjectVersion"
+                Resource: !Sub arn:aws:s3:::${RootStackTemplateConfigBucket}/*
+  Pipeline:
+    Type: "AWS::CodePipeline::Pipeline"
+    Properties:
+      ArtifactStore:
+        Location: !Ref ArtifactStore
+        Type: S3
+      RestartExecutionOnUpdate: false
+      RoleArn: !GetAtt CodePipelineIAMRole.Arn
+      Stages:
+        - Name: Source
+          Actions:
+            - Name: GitHub
+              ActionTypeId:
+                Category: Source
+                Owner: ThirdParty
+                Provider: GitHub
+                Version: 1
+              Configuration:
+                Owner: PRX
+                Repo: publish.prx.org
+                Branch: master
+                OAuthToken: !Ref GitHubOAuthToken
+              OutputArtifacts:
+                - Name: GitHubSourceArtifact
+              RunOrder: 1
+        - Name: Build
+          Actions:
+            - Name: Dockerize
+              ActionTypeId:
+                Category: Build
+                Owner: AWS
+                Provider: CodeBuild
+                Version: 1
+              Configuration:
+                ProjectName: !Ref DockerBuildProject
+              InputArtifacts:
+                - Name: GitHubSourceArtifact
+              OutputArtifacts:
+                - Name: DockerBuildArtifact
+              RunOrder: 1
+  DockerBuildProject:
+    Type: "AWS::CodeBuild::Project"
+    Properties:
+      Artifacts:
+        Type: CODEPIPELINE
+      Description: Tests, builds, and publishes a Docker image
+      Environment:
+        ComputeType: BUILD_GENERAL1_SMALL
+        EnvironmentVariables:
+          - Name: AWS_ACCOUNT_ID
+            Value: !Ref AWS::AccountId
+          - Name: TEMPLATE_CONFIG_BUCKET
+            Value: !Ref RootStackTemplateConfigBucket
+          - Name: TEMPLATE_CONFIG_ARTIFACT
+            Value: !Ref RootStackTemplateConfigArchive
+          - Name: ECR_REGION
+            Value: ${ECRRegion}
```

Comment:
I'm not sure this syntax would work here; might have been a typo in audiogram. Switch to `!Ref` to be safe